TestResultSummaryService/parsers/BenchmarkMetric.js

```diff
@@ -1,63 +1,87 @@
-// Note: The correct value for the metric must always reside in indice 1 (second element) of the regex output array
+// Note: The correct value for the metric must always reside in indice 1 (capture group) of the regex output array
+// use outerRegex when there are any warmup/cold runs before measure run that you do not wish to include.
+// if not familiar with regex look at other benchmarks & use https://regex101.com/ :)
 
+/* Format of each Benchmark
+ * 	Benchmarkname: {
+ * 	outerRegex:  Regex with capture group: all text after warmup runs is fine
+ * 
+ *  metrics:
+ *  [  // array of metrics
+ * 		{ //individual metric
+ * 			name: name of the metric
```

Comment:
Let's fix indentation here and other places. @llxia Are we following any convention for tabs vs spaces for TRSS? PerfNext seems to be using both.